### DISCUSSION TITLE
How to disable Nitro /server/api in front?

### DISCUSSION THREAD

**Discussion by JosukeH** [nuxt/nuxt#18719] [Questions]
I want to disable the api in front to only be able to be  running in my fetch 

example i can get:

localhost/api/hello

and get the content of my api.

**Comment by Baroshem** [comment]
I dont think it is that easily possible but maybe you could add a simple query to your endpoint with some secret value that you will only be able to access the endpoint by passing the correct secret? Otherwise you could return a 403 error. 

Then you could add it to your useFetch as query and then you will be able to get the data from it while not being so accessible in the browser.

**Comment by TheAlexLichter** [comment]
What you can do is fully pre-rendering your website via `nuxi generate`. If you do not do that and use a server, disabling the API will lead to issues because you can't fetch data then.
